Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:alpine as node
+
+WORKDIR /app
+COPY package.json .
+RUN npm install --only=prod
+COPY . .
```

Comment:
Line 4 copies only package.json, then line 6 copies everything (including package.json)
Couldn't you delete line 4 and replace it with line 6?

Reply:
Its good practice to copy package.json before installing the node_modules the adding the src code!!

Reply:
I'm happy to be proved wrong, can you provide some documentation that confirms this so the next person knows why as well?

The copy's you're doing *seem* incorrect, so I'm sure others will wonder the same thing when they see it.

Reply:
This [ref](https://gist.github.com/dweinstein/9468644) might help.

Reply:
This is a bad source. As far as I can tell, it's some guy writing code, saying it's good. There's no reasoning or explanation, and this guy doesn't seem to be an authoritative source.